assets/js/customizer-preview.js

```
@@ -0,0 +1,32 @@
+ * If you are looking for production-ready output files, see mode: "production" (https://webpack.js.org/configuration/mode/).
+ */
+/******/ (() => { // webpackBootstrap
+/******/ 	var __webpack_modules__ = ({
+
+/***/ "./src/admin/customizer-preview.js":
+/*!*****************************************!*\
+  !*** ./src/admin/customizer-preview.js ***!
+  \*****************************************/
+/***/ (() => {
+
+eval("(function ($) {\n  wp.customize.section('trademate_out_of_stock', function (section) {\n    section.expanded.bind(function (isExpanded) {\n      if (isExpanded) {\n        // Replace with the actual Shop page URL dynamically passed from PHP\n        var shopPageUrl = tm_wp_customizer_settings.shopPageUrl;\n\n        // Redirect the Customizer preview to the Shop page\n        if (shopPageUrl) {\n          wp.customize.previewer.previewUrl.set(shopPageUrl);\n        }\n      }\n    });\n  });\n})(jQuery);\n\n//# sourceURL=webpack://trademate/./src/admin/customizer-preview.js?");
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)